src/fswAlgorithms/pointCloudProcessing/pointCloudTriangulation/pointCloudTriangulation.h

```diff
@@ -79,7 +79,7 @@ class PointCloudTriangulation: public SysModel {
     Eigen::Matrix3d dcm_C2C1; //!< [-] direction cosine matrix (DCM) from camera frame C1 to camera frame C2
     bool valid; //!< [-] validity flag for point cloud triangulation
     int pointCloudSize{}; //!< [-] number of points in point cloud
-    std::vector<Eigen::Vector3d> measuredPointCloud{}; //!< [-] measured point cloud
+    Eigen::MatrixXd measuredPointCloud; //!< [-] measured point cloud
```

Comment:
:large_orange_diamond: Same here on the default initializer.

Reply:
Same as above